components/x-teaser-timeline/src/TeaserTimeline.jsx

```diff
@@ -69,10 +69,10 @@ const TeaserTimeline = (props) => {
 										</li>
 									)
 								} else if (typeof item === 'string') {
-									return <li key="custom-slot" dangerouslySetInnerHTML={{ __html: item }} />
+									return <li key={`"custom-slot-${index}"`} dangerouslySetInnerHTML={{ __html: item }} />
```

Comment:
Tiny nitpick, I think this can be updated to remove the `"` as these will be included literally in the computed string value.

```suggestion
									return <li key={`custom-slot-${index}`} dangerouslySetInnerHTML={{ __html: item }} />
```

Same for L75.